contracts/payloads/IGP86/PayloadIGP86.sol

```diff
@@ -292,6 +298,59 @@ contract PayloadIGP86 is PayloadIGPMain {
             setVaultLimits(VAULT_weETH_ETH_AND_wsETH); // TYPE_2 => 74
     }
 
+    // @notice Action 8: Increase Caps on sUSDe-USDT, USDe-USDT & USDC-USDT DEX Pools 
+    function action8() internal isActionSkippable(8){
+        {
+            address USDC_USDT_DEX = getDexAddress(2);
+                {
+                    // Set max borrow shares
+                    IFluidDex(USDC_USDT_DEX).updateMaxBorrowShares(
+                        28_000_000 * 1e18
+                    );
+                }
+        }
+
+        {
+            address sUSDe_USDT_DEX = getDexAddress(15);
+                {
+                    // Set max sypply shares
+                    IFluidDex(sUSDe_USDT_DEX).updateMaxSypplyShares(
+                        30_000_000 * 1e18
+                    );
+                }
+        }
+
+        {
+            address USDe_USDT_DEX = getDexAddress(18);
+                {
+                    // Set max supply shares
+                    IFluidDex(USDe_USDT_DEX).updateMaxSupplyShares(
+                        12_500_000 * 1e18
+                    );
+                }
+        }
+    }
+
+    // @notice Action 9: Adjust CF, LT, LML for rsETH Vaults
+    function action9() internal isActionSkippable(9){
+        {
+            address rsETH_ETH__wstETH = getVaultAddress(78);
+            address rsETH_wstETH = getVaultAddress(79);
+
+            uint256 CF = 94 * 1e2;
+            uint256 LT = 96 * 1e2;
+            uint256 LML = 97 * 1e2;
+
+            IFluidVaultT1(rsETH_ETH__wstETH).updateLiquidationMaxLimit(LML);
```

Comment:
should use proper interface here if we have even though function signature should be the same. IFluidVault or IFluidVaultT2 if available

Reply:
I think IFluidVaultT2 is not available
Thrilok had suggested to use T1 when we increased the limit for one of the older T2 vaults